custom/icds/tasks/__init__.py

```diff
@@ -9,9 +9,9 @@
 from corehq.blobs.models import BlobMeta
 from corehq.sql_db.util import get_db_aliases_for_partitioned_query
 from corehq.util.datadog.gauges import datadog_counter
-from custom.icds.tasks.hosted_ccz import setup_ccz_file_for_hosting
 
 if settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS:
+    from custom.icds.tasks.sms import send_monthly_sms_report
```

Comment:
So, if the conditional task is defined in this file it gets registered in celery and if it's defined in the tasks.py, it doesn't, yeah?

Reply:
If so, might as well move that whole task to this file?

Reply:
yes, if its defined in this file its registered, but if its defined in `tasks/sms.py` it needs to be imported here since its not called anywhere else.

The reason behind keeping different tasks in different files is to keep them separate.

Reply:
Ok, makes sense, the tasks needs to be defined in tasks.py which it wasn't previously. So you can do the same in tasks.py which is the standard way we declare tasks.

Reply:
ya, there are two ways, either make tasks.py or a module `tasks` with `__init__` and then import them there.